proposals/lip-0045.md

```diff
@@ -129,8 +135,9 @@ We define the following constants:
 | Name                                          | Type   | Value                | Description                                                      |
 |-----------------------------------------------|--------|----------------------|------------------------------------------------------------------|
 | **Interoperability Constants**                |        |                      |                                                                  |
-| `MODULE_ID_INTEROPERABILITY`                  | uint32 | 64                   | ID of the Interoperability module.                               |
-| `MAINCHAIN_ID`                                | uint32 | 1                    | Chain ID of the Lisk mainchain.                                  |
+| `MODULE_ID_INTEROPERABILITY`                  | bytes  | TBD                  | ID of the Interoperability module.                               |
+| `OWN_CHAIN_ID`                                | bytes  | 0x00000000           | Local chain ID of the chain.                                     |
+| `MAINCHAIN_ID`                                | bytes  | 0x00000001           | Chain ID of the Lisk mainchain.                                  |
```

Comment:
I think it would be nice to follow the same pattern here as for other constants, by starting with the type of constant and then giving the specific name, i.e., here `CHAIN_ID_MAINCHAIN`. Similarly, below `CHAIN_NAME_MAINCHAIN` or `NETOWRK_ID_MAINCHAIN`.

Reply:
updated